recipes/bob.extension/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON -B setup.py install --single-version-externally-managed --record record.txt
```

Comment:
This might make sense for this specific package but since these are automated pulls and starting from the next one they will have custom code in `build.sh`. So can we keep it for consistency with future `bob` packages? I am also waiting for [this](https://github.com/conda/conda-build/pull/859) to be merged to be able to package the rest of the `bob` packages.